apis/opts/mountpoint.go

```diff
@@ -10,11 +10,7 @@ import (
 
 // CheckBind is used to check the volume bind information.
 func CheckBind(b string) ([]string, error) {
-	if strings.Count(b, ":") > 2 {
```

Comment:
Why delete the `:` numbers check ?

Reply:
This is same as using the following code `arr := strings.SplitN(b, ":", 3)`. @HusterWan